rfcs/0055-retired-committers.md

```diff
@@ -0,0 +1,134 @@
+---
+feature: retired-committers
+start-date: 2019-08-25
+author: Till Höppner
+co-authors: Graham Christensen
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Many people were given push access to the nixpkgs repository, which is kept even if
+these committers become inactive. This RFC proposes moving these contributors to
+a new team without push access.
+
+# Motivation
+[motivation]: #motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+Each committer represents secrets and access which need to be managed carefully.
+These come in the form of passwords, SSH and GPG keys, and leaking them can put nixpkgs
+at risk of of unauthorized modification.
+
+Because every secret with push access can be leaked, we should keep their number as low as necessary,
+here by deactivating the push access of inactive committers.
```

Comment:
One motivation could be as well that they might have also loose access to their account.

Reply:
Good point. Should I include that in the motivation?

Reply:
Yes.